docs/routing.rst

```diff
@@ -90,6 +90,28 @@ under-score is encountered. As an example:
 Without this sanitation it would e.g. be impossible to implement an
 [OData](http://www.odata.org) API.
 
+Parameter Variable Converters
+-----------------------------
+
+Connexion supports Flask's ``int``, ``float``, and ``path`` route parameter
+variable converters. Specify a route parameter's type as ``integer`` or
+``number`` or its type as ``string`` and its format as ``path`` to use these
+converters. For example:
+
+.. code-block:: yaml
+
+  paths:
+    /greeting/{name}:
+      # ...
+      parameters:
+        - name: name
+          in: path
+          required: true
+          type: string
+          format: path
+
+will create an equivalent Flask route ``/greeting/<path:name>``.
```

Comment:
I would add one sentence on the implications, i.e. that the greeting name (in this example) can now contain forward slashes.